src/test/java/org/jenkinsci/plugins/docker/workflow/declarative/DeclarativeDockerUtilsTest.java

```diff
@@ -65,6 +65,8 @@ public static void setup() throws Exception {
 
     @Test
     public void plainSystemConfig() throws Exception {
+        // TODO: Fails using Windows Git
+        DockerTestUtil.assumeNotWindows();
```

Comment:
nit: I'm not totally sure why `DockerTestUtil.assumeNotWindows` exists, but if you use `assumeFalse` directly you can have the message show up when the test is skipped (same goes for `DockerBasicModelDefTest.failureBeforeStages`, and if you want to make this change I think you'll need to add some imports manually):

```suggestion
        assumeFalse("Fails using the version of Git installed on the Windows ACI agents on ci.jenkins.io", Functions.isWindows());
```